src/libraries/shims/ApiCompat.proj

```diff
@@ -5,7 +5,7 @@
     <BaselineApiCompat Condition="'$(BaselineApiCompat)' == ''">false</BaselineApiCompat>
     <PreviousNetCoreApp>net5.0</PreviousNetCoreApp>
```

Comment:
It seems like we need to update this? We are hitting the error that checks the tfm exists in the ref pack we are comparing against.